smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/PackageManifestGenerator.kt

```diff
@@ -76,14 +70,8 @@ class PackageManifestGenerator(
         dependency: SymbolDependency,
     ) {
         writer.openBlock(".package(", "),") {
-            val scope = dependency.getProperty("scope", String::class.java).getOrNull()
-            scope?.let {
-                writer.write("id: \$S,", "$it.${dependency.packageName}")
-            }
-            val url = dependency.getProperty("url", String::class.java).getOrNull()
-            url?.let {
-                writer.write("url: \$S,", it)
-            }
+            val url = dependency.getProperty("url", String::class.java).get()
+            writer.write("url: \$S,", url)
```

Comment:
Note that the `url` property is guaranteed to succeed here since the method is only called on dependencies that have already been verified to have `url` set.